examples/plot_compare_reduction.py

```diff
@@ -42,14 +43,16 @@
         'classify__C': C_OPTIONS
     },
     {
-        'reduce_dim': [SelectKBest(chi2)],
+        'reduce_dim': [SelectKBest(chi2), SelectKBest(info_gain),
```

Comment:
Given that other feature selection scorers are absent here, I'm not sure this is the best place to show this off.

I also wonder whether there's room for an example comparing feature selection on a few different datasets, as in the Yang and Pedersen (1997) paper you cite.